.devcontainer/devcontainer.json

```diff
@@ -62,7 +62,7 @@
         "json.schemas": [
           {
             "fileMatch": ["homeassistant/components/*/manifest.json"],
-            "url": "./script/json_schemas/manifest_schema.json"
+            "url": "/workspaces/HAcore/script/json_schemas/manifest_schema.json"
```

Comment:
That won't work. For me it's `/workspaces/core/...´. For someone else it might be different...

Reply:
Messed up my first PR here.. :$
Apparently there is ${containerWorkspaceFolder}, which is hard to find.. Changing the PR